external_executors/local_cromwell/local_cromwell.py

```diff
@@ -218,6 +269,9 @@ def localize_inputs(obj, match_func, localize_func):
     f.write(script)
 END
 
+# Set a maximum run time of 90 minutes to kill stalled jobs.
```

Comment:
This probably deserves a comment in whatever documentation you have for this. I imagine it'd be pretty rare to run this on at-scale data, but it'd be nice to tell a user who might might this up and use it for something weird that it won't work for that.

Reply:
Thanks, added to the readme.